src/screens/Settings/ContentAndMediaSettings.tsx

```diff
@@ -120,7 +121,15 @@ export function ContentAndMediaSettingsScreen({}: Props) {
                 name="show_trending_topics"
                 label={_(msg`Enable trending topics`)}
                 value={!trendingDisabled}
-                onChange={value => setTrendingDisabled(!value)}>
+                onChange={value => {
+                  const hide = Boolean(!value)
+                  if (hide) {
+                    logEvent('trendingTopics:hide', {context: 'settings'})
+                  } else {
+                    logEvent('trendingTopics:show', {context: 'settings'})
+                  }
```

Comment:
Realized we missed adding `show` previously, so moved this here to simplify both directions. No other "show" actions exist in the app rn.